NimbleTests/Matchers/BeIdenticalToTest.swift

```diff
@@ -1,36 +0,0 @@
-import XCTest
```

Comment:
Is it possible to just `#if !os(Linux)` this entire file? I'd still prefer to have tests for iOS / OS X :wink: